src/UNL/MediaHub/DefaultHomepage.php

```diff
@@ -49,6 +51,11 @@ function getTopMedia()
                 break;
             }
 
+            if (strtotime($media->datecreated) < $after_date) {
+                //We only want media that was added in the last 4 months
```

Comment:
fixed in https://github.com/unl/UNL_MediaHub/pull/331/commits/2ab10d9956dfcd461ecae8c0fcf7f15c8a75685e